canonical/lxd-ui
# Show 404 in all project subpages if project does not exist [WD-4068]

## Done

- When visiting the `/:project` URL, or any `/:project/` subpage, show the 404 if the project does not exist.

Fixes [WD-4068](https://warthogs.atlassian.net/browse/WD-4068)

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - Visit [a bogus project URL](https://lxd-ui-351.demos.haus/ui/bogusproject), or any of its subpages
    - Check that the 404 is shown

[WD-4068]: https://warthogs.atlassian.net/browse/WD-4068?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
